common/perf-test-core/pom.xml

```diff
@@ -46,7 +46,7 @@
           <rules>
             <bannedDependencies>
               <includes combine.children="append">
-                <include>com.fasterxml.jackson.core:jackson-databind:[2.13.1]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
+                <include>com.fasterxml.jackson.core:jackson-databind:[2.13.2.1]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
```

Comment:
should this be 2.13.2 and not 2.13.2.1 ? 

https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind